common/api/appui-react.api.md

```diff
@@ -3705,7 +3705,7 @@ export enum SelectionScope {
 }
 
 // @public
-export const SelectionScopeField: ConnectedComponent<typeof SelectionScopeFieldComponent, Omit_3<SelectionScopeFieldProps, "availableSelectionScopes" | "activeSelectionScope">>;
+export const SelectionScopeField: ConnectedComponent<typeof SelectionScopeFieldComponent, Omit_3<SelectionScopeFieldProps, "activeSelectionScope" | "availableSelectionScopes">>;
```

Comment:
Try reverting that specific change, we've had issues with that declaration for a long time, where it does something in one place and a different one in a different place. (This is usually "solved" by going directly to that package folder,
* `npm run clean`
* `npm run build`
* `npm run extract-api`

Reply:
Thanks, fixed.